examples/ws-server/client.node.ts

```diff
@@ -2,7 +2,6 @@ import {
     NetworkPipe,
 } from '../../src/types';
 
-import Platform from '../../src/Platform';
```

Comment:
is client.node.ts unable to run?  Should I simply import ../../src/node/Platform?

Reply:
I think I forgot to change this import. you can do import Platform from "../../#{platform}/Platform"

Reply:
sorry its #{target} not #{platform}